### ISSUE TITLE
How can I run a simple example with this framework?

### ISSUE THREAD

**Issue by linlinge** [aboulch/snapnet#20]
I want to run a simple example with this framework, that means I input a point cloud and the output is the label file. But I was confused how to do this.  The scripts "sem3d_gen_images.py", "sem3d_test_backproj.py" and "sem3d_train.py" seem like they will load some models to train a network. So, how to use the pre-trained weights to run a simple example?  Thank you very much!

**Comment by aboulch** [comment]
Hello,
- The `sem3d_gen_images.py` script should not load a new model.
- I think you can apply the procedure to train on your file if it is in the Semantic3D format. You just need to change the filenames in the scripts accordingly.
- You still can apply the pretrained weights but you need to generate the images before.
